2025-06-30 14:03:08,846 - modules.base.BaseAPIClient - ERROR - Request failed: HTTPSConnectionPool(host='your-opensilex-instance.com', port=443): Max retries exceeded with url: /security/authenticate (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000203AEF4F8F0>: Failed to resolve 'your-opensilex-instance.com' ([Errno 11001] getaddrinfo failed)"))
2025-06-30 14:03:08,846 - modules.base.BaseAPIClient - ERROR - Authentication error: Request failed: HTTPSConnectionPool(host='your-opensilex-instance.com', port=443): Max retries exceeded with url: /security/authenticate (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000203AEF4F8F0>: Failed to resolve 'your-opensilex-instance.com' ([Errno 11001] getaddrinfo failed)"))
